static/src/components/projectCard.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState, useEffect} from 'react';
+import { Link } from 'gatsby';
+import { Line } from '../utilities';
+import styled from 'styled-components';
+
+const ProjectCard = styled.div`
+  margin: 0.5rem auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minMax(200px, auto));
+  grid-gap: 1rem;
+  align-items: center;
+  a {
+    text-align: center;
+  }
+  h2 {
+    margin-bottom: 0.1rem;
+  }
+  ul {
+    margin: 0;
+  }
+`;
+
+const getProjectCard = ({project_name, org_url, po_name, description, description_link}) => (
+  <ProjectCard key={project_name}>
+    <div>
```

Comment:
Might be able to clean up this extra div here.

Reply:
Need this div for formatting.